environment/environment.go

```diff
@@ -14,6 +14,10 @@ const (
 	commaSeparator = ","
 )
 
+var (
+	ErrNotFound = errors.New("Environment variable not found")
+)
+
 // GetLogLevel returns the loglevet or panics. This is called before any logger
 // is available. i.e. don't use a logger here.
```

Comment:
We have previously established the OrFatal convention because some people object to functions which terminate. I understand we cant use logger.Fatalf here because this must be called before it is created. I think OrPanic would be better aligned with established patterns.

Personally, I have no problem with declaring that the environment module is 'allowed to panic or fatal out' because it is designed to be used as part of pre-flight startup. And then remove _all_ the OrWhatever suffixes.

But we should get everyone on board with that before doing so.

Reply:
Yes this is a speculative change only and may go away...